config/globalVariable.js

```diff
@@ -89,24 +89,24 @@ module.exports = function () {
   // Load all message constants
   global.messageConstants = {};
 
-  fs.readdirSync(ROOT_PATH + '/generics/messageConstants').forEach(function (file) {
+  fs.readdirSync(path.resolve('generics/messageConstants')).forEach(function (file) {
     if (file.match(/\.js$/) !== null) {
       let name = file.replace('.js', '');
-      global.messageConstants[name] = require(ROOT_PATH + '/generics/messageConstants/' + file);
+      global.messageConstants[name] = require(path.resolve('generics/messageConstants' , file));
     }
   });
 
   // Load all kafka consumer files
-  fs.readdirSync(ROOT_PATH + '/generics/kafkaConsumers/').forEach(function (file) {
+  fs.readdirSync(path.resolve('generics/kafkaConsumers/')).forEach(function (file) {
     if (file.match(/\.js$/) !== null) {
       var name = file.replace('Consumer.js', '');
-      global[name + 'Consumer'] = require(ROOT_PATH + '/generics/kafkaConsumers/' + file);
+      global[name + 'Consumer'] = require(path.resolve('generics/kafkaConsumers/' , file));
     }
   });
 
   global.sessions = {};
 
-  const libraryCategoriesHelper = require(MODULES_BASE_PATH + '/library/categories/helper');
+  // const libraryCategoriesHelper = require(MODULES_BASE_PATH + '/library/categories/helper');
```

Comment:
@Prajwal17Tunerlabs any particular reason why we commented this import?

Reply:
@VISHNUDAS-tunerlabs  `libraryCategoriesHelper` was not being used anywhere, so i commented it.